gmso/lib/potential_templates.py

```diff
@@ -149,16 +145,11 @@ def assert_can_parameterize_with(
                     f"parameters: {parameters}"
                 )
 
-    class Config:
-        """Pydantic configuration for potential template."""
-
-        allow_mutation = False
-        fields = {
-            "expected_parameters_dimensions_": "expected_parameters_dimensions"
-        }
-        alias_to_fields = {
-            "expected_parameters_dimensions": "expected_parameters_dimensions_"
-        }
+    # TODO[pydantic]: The following keys were removed: `allow_mutation`, `fields`.
```

Comment:
I think this TODO can get removed since the model_config seems to be the way to handle it in V2.

Reply:
agree